File: src/2037.minimum-number-of-moves-to-seat-everyone.2148/solution.ts
function minMovesToSeat (seats: number[], students: number[]): number {
    seats.sort((a, b) => a - b);
    students.sort((a, b) => a - b);
    let result = 0;
    for (let i = 0; i < seats.length; i++) {
        result += Math.abs(seats[i] - students[i]);
    }
    return result;
}

File: src/2037.minimum-number-of-moves-to-seat-everyone.2148/solution.java
class Solution {
    public int minMovesToSeat(int[] seats, int[] students) {
        Arrays.sort(seats);
        Arrays.sort(students);
        int result = 0;
        for(int i=0;i<seats.length;i++){
            result += Math.abs(seats[i]-students[i]);
        }
        return result;
    }
}